roles/elk/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+---
+- name: install elasticsearch
```

Comment:
These installs should all be merged into a single block. The notify isn't important because you have explicit steps later to start them.